### PR TITLE
search 7.4 | sidecar version changes | LRDOCS-10192

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-10192
cc @lipusz

### CODE REVIEW

File: docs/dxp/latest/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/installing-elasticsearch.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 Here's how to install, configure, and start Elasticsearch on-premises.
 
 ```{important}
-The Sidecar Elasticsearch server bundled with Liferay 7.3+ uses the Elasticsearch OSS distribution. Do not install the OSS version for production. To run Liferay securely with Elasticsearch, you must install the Basic level of Elasticsearch at a minimum. See [Elastic's subscriptions page](https://www.elastic.co/subscriptions) for more information.
+The Sidecar Elasticsearch server bundled with Liferay 7.3 and Liferay 7.4 prior to DXP U17+ and Portal GA17+ uses the Elasticsearch OSS distribution. Do not install the OSS version for production. To run Liferay securely with Elasticsearch, you must install the Basic level of Elasticsearch at a minimum. See [Elastic's subscriptions page](https://www.elastic.co/subscriptions) for more information.
```
[the remark this answers is not in the document]
Comment (lipusz) [reply docs/dxp/latest/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/installing-elasticsearch.md:6]: >Liferay 7.4 prior to DXP U17+ and Portal GA17

sounds weird to me. (Sorry, my OCD. :innocent: )

[the remark this answers is not in the document]
Comment (rbohl) [reply docs/dxp/latest/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/installing-elasticsearch.md:6]: Well, I think then I'd need to say
> ...with Liferay 7.3 and prior to Liferay 7.4 DXP U17+ and Liferay Portal GA17+... 

This sounds more confusing to me--I like saying "In this major version prior to these minor versions" better than "prior to this major and minor version and this major and minor version" but it's a minor preference, so I can change it.

[the remark this answers is not in the document]
Comment (lipusz) [reply docs/dxp/latest/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/installing-elasticsearch.md:6]: >The Sidecar Elasticsearch server bundled with Liferay 7.3 and early Liferay 7.4 versions (**prior to Liferay DXP 7.4 U17/Liferay Portal 7.4 GA17**) uses the Elasticsearch OSS distribution. ...

I believe "prior to" and removing the "+" is more accurate as starting with U17/GA17 it is no longer OSS.

[the remark this answers is not in the document]
Comment (lipusz) [reply docs/dxp/latest/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/installing-elasticsearch.md:6]: Or say until GA16/U16.

[the remark this answers is not in the document]
Comment (rbohl) [reply docs/dxp/latest/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/installing-elasticsearch.md:6]: > The Sidecar Elasticsearch server bundled with Liferay 7.3 and Liferay 7.4 versions through Liferay DXP 7.4 U16/Liferay Portal 7.4 GA16 uses the Elasticsearch OSS distribution. ...

**Comment by lipusz** [comment]
Hi Guys,

Do you have any news on this? Thanks!

**Comment by jrhoun** [comment]
Merged. Thanks!

**Pull request closed without merging**